index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <div id="main">
+      <h1 id="title">Steve Jobs</h1>
+      <div id="img-div">
+        <p id="img-caption">The eccentric innovator who brought us the iPhone</p>
+        <img id="image" src="jobs.jpeg" alt="steve jobs pic" />
+      </div>
+
+      <ul id="tribute-info">
+        <li>1955 - Born in San Francisco, CA</li>
+        <li>1972 - Attends Reed College</li>
+        <li>1974 - Drops out and travels to India</li>
+        <li>1976 - Founded Apple</li>
+        <li>1977 - Releases the Apple II</li>
+        <li>1983 - Releases the Apple Lisa</li>
+        <li>1984 - Releases the Apple Machintosh</li>
+        <li>1985 - Gets Fired from Apple</li>
+        <li>1985 - Founded Next</li>
+        <li>1986 - Takes over Pixar</li>
+        <li>1995 - Produces Toy Story</li>
+        <li>1996 - Apple buys Next for $427 million dollars and merges with it</li>
+        <li>1997 - Back as CEO of Apple</li>
+        <li>2001 - Releases the iPod</li>
+        <li>2007 - Releases the iPhone</li>
+        <li>2010 - releases the iPad</li>
+        <li>2011 - Dies at the age of 56 in Palo Alto, CA</li>
+      </ul>
```

Comment:
Since you may want to target different elements within your list as well as the list as a whole, it is better to use a CSS class instead of an id

Reply:
- [ ] OPTIONAL Kindly notice that it very important to give a clear reason of why you requested changes.